From bf29282bea5b190655841a5bce650599e48967a0 Mon Sep 17 00:00:00 2001
From: Staroselskii Georgii <gosha371@gmail.com>
Date: Tue, 21 Jun 2016 18:57:01 +0300
Subject: [PATCH 2/3] Makefile: remove checking

---
 Makefile | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/Makefile b/Makefile
index e0e78f0..c0e21fd 100644
--- a/Makefile
+++ b/Makefile
@@ -1080,13 +1080,3 @@ clean:
 	rm -fr *.mod.c *.mod *.o .*.cmd *.ko *~
 	rm -fr .tmp_versions
 endif
-
-check:
-	vagrant up || vagrant init ubuntu/vivid32; vagrant up
-	TMP=$$(vagrant ssh-config | grep Port | cut -d ' ' -f 4);\
-	ssh -p $$TMP -i $(HOME)/.vagrant.d/insecure_private_key vagrant@localhost 'bash -s' < fetch.sh
-	vagrant ssh -c 'sudo dpkg -i linux-headers-4.0.1-040001_4.0.1-040001.201504290935_all.deb linux-headers-4.0.1-040001-generic_4.0.1-040001.201504290935_i386.deb linux-image-4.0.1-040001-generic_4.0.1-040001.201504290935_i386.deb'
-	vagrant ssh -c 'sudo apt-get update && sudo apt-get install --yes git build-essential'
-	vagrant ssh -c 'if ! [ -d rtl8812AU_8821AU_linux ] ; then git clone https://github.com/abperiasamy/rtl8812AU_8821AU_linux.git; fi'
-	vagrant ssh -c 'cd rtl8812AU_8821AU_linux; make KVER=4.0.1-040001-generic'
-	vagrant ssh -c 'cd rtl8812AU_8821AU_linux; sudo make install'
-- 
2.5.0

